.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches:
+    - master
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 40
+    strategy:
+      fail-fast: false
+      matrix:
+        java:
+        - 11
```

Comment:
I don't think it's really necessary to run build on many JVM versions

Reply:
OK, removed